documentation-src/metalsmith/content/reference.md

```diff
@@ -13,6 +13,8 @@ during the search cycle with events:
  - `result`: when the results are retrieved from Algolia
  - `error`: when Algolia sends back an error
  - `searchQueueEmpty`: when there is no more on going searches
```

Comment:
I'm seeing this now, while it was made earlier, but this should be `ongoing` or `pending`